Modules/ReportBuilder/Services/ReportRenderer.php

```
@@ -0,0 +1,337 @@
+    }
+
+    /**
+     * Wrap content in HTML template with PDF styles.
+     *
+     * @param string         $content  The rendered content
+     * @param ReportTemplate $template The template
+     * @param mixed          $invoice  The invoice data
+     *
+     * @return string Complete HTML document
+     */
+    private function wrapInHtmlTemplate(string $content, ReportTemplate $template, $invoice): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$template'. (undefined)

(UnusedFormalParameter)